public/locales/en/translation.json

```diff
@@ -13,5 +13,23 @@
   },
   "ar": {
     "1": "Arabic"
+  },
+  "blogposts": {
+    "blog0": {
```

Comment:
this should be an array, not blog0 blog1 blog2 in this case because it's simply a collection of blog items